superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.test.tsx

```diff
@@ -321,7 +321,8 @@ test('validates the pre-filter value', async () => {
   ).toBeInTheDocument();
 });
 
-test("doesn't render time range pre-filter if there are no temporal columns in datasource", async () => {
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("doesn't render time range pre-filter if there are no temporal columns in datasource", async () => {
```

Comment:
Hey @michael-s-molina just out of curiosity, what's the reason for skipping this test?

Reply:
For a time now I have noticed that this test is failing every now and then. So I'm disabling it until we fix it.